### ISSUE TITLE
Development status, updates and future of KeeFox

### ISSUE THREAD

**Issue by atomtigerzoo** [kee-org/KeeFox#728]
Hi @luckyrat and everyone else :)
To me it seems like development is stalled - sorry for the harsh words. Over last months nothing happened and it seems like the author is not responding to issues/PRs. Not even the important ones. And that is indeed very sad to see! KeeFox is great plugin and it deserves support!

So I'm asking, without any offense, how can we see the future of this addon? What about fixing bugs? What are the plans and how can we get on with fixes, bugs and new development?

Maybe it is possible to get some more authors into the team that can review and merge PRs or even just fix small bugs?

It would be great if we could get this plugin to new life :)
Thank you for your work so far and all the time you invested! 👍

**Comment by xchs** [comment]
> how can we see the future of this addon?

I've just read this: http://keefox.org/news/detail/2015/10/18/keefox-in-2016-and-beyond

It would be a pity and a great loss if KeeFox would actually abandoned sooner or later!

**Comment by dlech** [comment]
If anyone has loads of spare time and wants to convert KeeFox to a WebExtension, here is the latest info: https://blog.mozilla.org/addons/2016/11/23/add-ons-in-2017/

**Comment by luckyrat** [comment]
Nothing significant has changed in the past year apart from a firmer date of end of 2017 for the removal of support for the add-on technology that KeeFox is currently based upon. That's a little later than first announced and a few versions earlier than I would expect but no huge change.

Although WebExtensions has improved, it still won't be able to offer the same level of seamless integration that KeeFox currently offers and there is still no viable migration path from the current add-on technology to WebExtensions... although I hope that will change next year.

In any case, there is still a tonne of work to do to keep KeeFox going into the future and just focussing on bug fixes won't get us very far. That said, if there are important bugs (or security issues as per the previous weekend) I'll try to get them fixed as soon as I can and if anyone wants to submit PRs for such issues I would review and merge them.

I have already started experimenting with a conversion to WebExtensions but it's early days and I won't have anything tangible to show for at least a few months. Perhaps when I have completed the essential parts of it, I can put it onto GitHub on a development branch and welcome PR contributions to speed along the process.
